packages/graphiql/src/components/__tests__/GraphiQL.spec.tsx

```diff
@@ -98,6 +98,31 @@ describe('GraphiQL', () => {
     expect(secondCalled).toEqual(true);
   });
 
+  it('should refetch schema after a fetchError', async () => {
```

Comment:
The test description sounds like the refetch would be automatically triggered after a fetch error. Can we make it more clear that the refetch has to be triggered "manually" (i.e. providing a different fetcher)

Reply:
👍  How does

> should refresh schema with new fetcher after a fetchError

 sound?

Reply:
Sounds good!